opensrs/opensrsapi.py

```diff
@@ -405,6 +411,8 @@ def register_domain(self, domain, purchase_period, user, user_id,
                             'Invalid syntax on domain')):
                     raise errors.InvalidDomain(e)
                 raise errors.DomainRegistrationFailure(e)
+            if e.response_code == self.CODE_CLIENT_TIMED_OUT:
+                raise errors.DomainRegistrationTimedOut(e)
```

Comment:
We definitely shouldn't handle `SSLError` here. I just wanted to say that we might add some special error which won't be used in our DS.